server/db/data/recipedia.schema

```diff
@@ -9,10 +9,51 @@ type Recipe {
   name
   rating
   contains
+  servings
+  calories
+  total_fat
+  saturated_fat
+  cholesterol
+  sodium
+  potassium
+  total_carbohydrates
+  dietary_fiber
+  protein
+  sugars
+  vitamin_a
+  vitamin_c
+  calcium
+  iron
+  thiamin
+  niacin
+  vitamin_b6
+  magnesium
+  folate
 }
 
 # Define Directives and index
 xid: string @index(exact) .
```

Comment:
Do you plan to do the field renaming in this PR or a future PR? I think it's fine to do this later

Reply:
Do you think the schema and the .rdf need to use camelCase? I feel snake_case does look alright in the schema and .rdf file.

For example, having baking_soda rather than bakingSoda and having vitamin_c vs vitaminC.

Reply:
camelCase is preferred, and seems to be the dgraph style

Reply:
OK, I can do that. For example:

Reply:
Also, I think dgraph use snake_case in their tutorials. For example: https://tour.dgraph.io/schema/1/.

However, I agree it is nice to have it in camelCase so that it matches with our code base and ER diagram.

Reply:
Oh, I see! Well in that case snake_case is fine. I was mistaken